hack/verify-godeps.sh

```diff
@@ -64,8 +64,9 @@ _kubetmp="${_kubetmp}/kubernetes"
 
 # Do all our work in the new GOPATH
 export GOPATH="${_tmpdir}"
+export PATH="${GOPATH}/bin:${PATH}"
```

Comment:
You had raised concerns about doing stuff like this here: https://github.com/kubernetes/kubernetes/pull/59048#discussion_r165268522

I'm wondering if a method like https://github.com/kubernetes/kubernetes/pull/59210 would be better. This approach uses the run-in-gopath method so that the utility is built hermetically, and doesn't A) overwrite a version of the tool already existing in the user's environment or B) pollute their $GOPATH/pkg

Let me know what you think!

Reply:
The issue I took was changing global variables deep inside a function.

Changing them in a first-level script is OK.

Basically, any place that changes GOPATH should also change PATH to frontload the new GOPATH/bin

Reply:
I added a commit here that seems simple - PTAL?